code/modules/jobs/job_types/chaplain/chaplain_vorpal_scythe.dm

```diff
@@ -19,12 +19,11 @@ If the scythe isn't empowered when you sheath it, you take a heap of damage and
 	var/obj/item/vorpalscythe/scythe = active_item
 	if(!scythe)
 		return FALSE
-	if(scythe.empowerment < SCYTHE_SATED)
+	var/obj/item/bodypart/part = hand
+	if(scythe.empowerment < SCYTHE_SATED && part)
```

Comment:
make this an early return, like
```dm
	if(isnull(part) || (scythe.empowerment > SCYTHE_SATED))
		return ..()
	// the rest of the code where we do the to_chat and sound and damage goes here
```

this is just so the indentation looks a lot cleaner. we also do the `isnull()` first because that reads better on the shortcircuit imo